addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -313,3 +401,35 @@ func (r *MirrorPeerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&multiclusterv1alpha1.MirrorPeer{}, builder.WithPredicates(mpPredicate)).
 		Complete(r)
 }
+
+func (r *MirrorPeerReconciler) labelRBDStorageClasses(ctx context.Context, mp multiclusterv1alpha1.MirrorPeer, storageClusterNamespace string) []error {
+	// Get all StorageClasses in storageClusterNamespace
+	scs := &storagev1.StorageClassList{}
+	klog.Infof("Getting all StorageClasses in namespace %q", storageClusterNamespace)
+	err := r.SpokeClient.List(ctx, scs, client.InNamespace(storageClusterNamespace))
```

Comment:
StorageClass is cluster scoped resource. No need of namespace.

Reply:
ok